### PR TITLE
Release v0.2

### PR DESCRIPTION
**Pull request by bjlittle**
This PR includes bumps to latest release.

### CODE REVIEW

**Comment by bjlittle** [comment]
@lbdreyer Heads up ...

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ before_install:
 
 install:
     - |
-      MINICONDA_URL="http://repo.continuum.io/miniconda"
+      MINICONDA_URL="https://repo.continuum.io/miniconda"
```
Comment (ocefpaf) [inline .travis.yml]: You don't need to do that manually. You should update `conda-smithy` and rerender the feedstock. (Sorry if I missed this before and feel free to contact me when you need help in the feedstocks.)

Comment (lbdreyer) [reply .travis.yml]: Thanks for pointing this out @ocefpaf :smiley: 

@bjlittle I have used conda-smithy and have a PR up (see PR #2 ) that you will need to rebase onto

**Comment by lbdreyer** [comment]
Appveyor has finally finished!

Thanks for sticking with this @bjlittle!!

**Comment by bjlittle** [comment]
Hoo-rah! Awesome, thanks @lbdreyer and @ocefpaf !

**Pull request merged**